imports/plugins/core/accounts/client/components/InviteShopMemberDialog.js

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import i18next from "i18next";
+import SimpleSchema from "simpl-schema";
+import CloseIcon from "mdi-material-ui/Close";
+import Button from "@reactioncommerce/catalyst/Button";
+import TextField from "@reactioncommerce/catalyst/TextField";
+import Select from "@reactioncommerce/catalyst/Select";
+import useReactoForm from "reacto-form/cjs/useReactoForm";
+import muiOptions from "reacto-form/cjs/muiOptions";
+import { useMutation } from "@apollo/react-hooks";
+import { useSnackbar } from "notistack";
+import {
+  Box,
+  Grid,
+  CardActions,
+  CardHeader,
+  CardContent,
+  Dialog,
+  IconButton,
+  makeStyles
+} from "@material-ui/core";
+import inviteShopMemberMutation from "../graphql/mutations/inviteShopMember";
+
+const useStyles = makeStyles((theme) => ({
+  cardRoot: {
+    overflow: "visible",
```

Comment:
Same issues here, the Card header and card content need to wrapped in `Card` component and this class needs to be added so that the select options are not cut off.